Postfix/postfix/tests/pipe3.json

```diff
@@ -21,15 +21,14 @@
     "action": {
       "name": "sent",
       "outcome": "success",
+      "outcome_reason": "success",
       "target": "network-traffic"
     },
     "destination": {
-      "address": "example.org",
-      "domain": "example.org",
+      "address": "127.0.0.1",
+      "domain": "127.0.0.1",
```

Comment:
Do you know why the domain was replaced by an ip-address?

Reply:
Because `relay=` field here contains specific info, unlike `relay=` in https://github.com/SEKOIA-IO/intake-formats/blob/ab50051dc7ddcde7f3d278cc5ebe95893f765f06/Postfix/postfix/tests/pipe1.json